internal/commands/app/new_app_inputs.go

```diff
@@ -73,50 +73,16 @@ func (i *newAppInputs) resolveTemplateID(ui terminal.UI, client realm.Client) er
 		return err
```

Comment:
opt/nit: since it seems like template starter apps is going to be disabled in beta.7 still, we probably don't need this client call.

We should probably also comment out the flag in [here](https://github.com/10gen/realm-cli/blob/90c58fda935f6a9dedfd24294dfc0a332f0a2759/internal/commands/app/create.go#L93-L101) so people don't run in to this unnecessarily

Reply:
just pointing out, if we comment out the flag, then the rest of this func after the first if check is effectively dead code (in addition to removing `ui.Auto.Confirm` above as well).  easy one line comment to avoid all of this (and could then probably just not change the rest)

Reply:
hmm I thought the idea was that we would still allow users who want to opt in to using templates could specify the flag? happy to ditch this, but that was the assumption I was operating under.

Reply:
hmm if we wanted to do that, I think we'd need to rebase this PR with the changes on the master branch to get the pull and create commands.

However, as they are kinda rough around the edges, I'd opt to remove for now and add templates support in 2.1.0